handlers.js

```diff
@@ -155,7 +188,11 @@ async function handle_message(reply, content) {
     handler = handlers[content.split(" ")[0]];
     if (handler != undefined) {
         try{
-            let r = await handler(content.split(" ").slice(1, 9));
+            var args = [content.split(" ").slice(1, 9)];
+            if (handler.name == reply_wait().name) {
```

Comment:
This feels a little funky to me - is there a cleaner way to approach this? I don't like the idea of hardcoding for this specific handler here if it can be avoided. Also, can you not just use `reply_wait.name` here?

Reply:
It is definitely funky..., `reply_wait.name` is just going to return "reply_wait". Adding the parentheses will return the enclosed `slow_function`.